modules/admin_manual/pages/configuration/server/request_tracing.adoc

```diff
@@ -0,0 +1,35 @@
+= Request Tracing
+
+ownCloud logs the `X-REQUEST-ID` header from clients in the ownCloud log when sent with client requests.
+
+The header helps when clients have a problem communicating with an ownCloud server, because:
+
+. The user can include the value in bug reports; and
+. System administrators can filter log files for the header value. 
+
+Storing this information makes searching more efficient, as system administrators don’t have to rely solely on normal log entry elements, such as timestamps and IP addresses.
+
+== The Header’s Value
+
+The header's value is a random string between 20 and 200 characters long.
+The string is randomly generated by the client and does not contain _any_ sensitive information.
+As a result it should not violate the user's privacy nor allow users to be tracked.
+
+== Required Server Configuration
+
+Before the value can be stored your web server's log files, your system administrator(s) need to configure two areas:
```

Comment:
can be stored in your web server's log files.

Reply:
Thanks for catching that omission, @pako81.